### ISSUE TITLE
SLURP spec of "Stay there." crashes GROne

### ISSUE THREAD

**Issue by ConstantineLignos** [VerifiableRobotics/LTLMoP#55]
Tested in the LTLMoP gumbo branch:
1. Open speceditor with the examples/slurptest/slurptest.spec spec.
2. Change the spec to only say "Stay there."
3. GROne crashes with the following stack:

<pre>
    Exception in thread "main" net.sf.javabdd.BDDException: ERROR: Cannot prime primed variables: 
        <sweep_done', sweep', p3', p6', p7', p4', p5', p8'>
        at edu.wis.jtlv.env.Env$JTLVBDDManagerPairing.prime(Env.java:3460)
        at edu.wis.jtlv.env.Env$JTLVBDDManagerPairing.access$3(Env.java:3457)
        at edu.wis.jtlv.env.Env.prime(Env.java:1735)
        at edu.wis.jtlv.env.module.Module.yieldStates(Module.java:552)
        at GROneGame.calculate_win(GROneGame.java:131)
        at GROneGame.<init>(GROneGame.java:87)
        at GROneGame.<init>(GROneGame.java:48)
        at GROneMain.main(GROneMain.java:114)
</pre>

**Comment by ConstantineLignos** [comment]
LTL generated:

``` text

-- LTL specification file
-- (Generated by the LTLMoP toolkit)

LTLSPEC -- Assumptions
    (
    TRUE & [](TRUE) & []<>(TRUE)
    );

LTLSPEC -- Guarantees
    (
        ([](((!s.sweep & next(s.sweep)) -> ( (next(s.bit0) <-> s.bit0) & (next(s.bit1) <-> s.bit1) & (next(s.bit2) <-> s.bit2) )))) & 
        ([]<>(( (next(s.bit0) <-> s.bit0) & (next(s.bit1) <-> s.bit1) & (next(s.bit2) <-> s.bit2) )))
&
            ( (!s.bit0 & !s.bit1 & !s.bit2) 
                 | (!s.bit0 & !s.bit1 & s.bit2)
                 | (!s.bit0 & s.bit1 & !s.bit2)
                 | (!s.bit0 & s.bit1 & s.bit2)
                 | (s.bit0 & !s.bit1 & !s.bit2)
                 | (s.bit0 & !s.bit1 & s.bit2)
            ) 

&
             []( ((!s.bit0 & !s.bit1 & !s.bit2)) -> ( ((!next(s.bit0) & !next(s.bit1) & !next(s.bit2)))
                                    | ((!next(s.bit0) & next(s.bit1) & next(s.bit2))) 
                                    | ((next(s.bit0) & !next(s.bit1) & !next(s.bit2)))  ) )  & 
             []( ((!s.bit0 & !s.bit1 & s.bit2)) -> ( ((!next(s.bit0) & !next(s.bit1) & next(s.bit2)))
                                    | ((!next(s.bit0) & next(s.bit1) & next(s.bit2))) 
                                    | ((next(s.bit0) & !next(s.bit1) & !next(s.bit2)))  ) )  & 
             []( ((!s.bit0 & s.bit1 & !s.bit2)) -> ( ((!next(s.bit0) & next(s.bit1) & !next(s.bit2)))
                                    | ((next(s.bit0) & !next(s.bit1) & !next(s.bit2))) 
                                    | ((next(s.bit0) & !next(s.bit1) & next(s.bit2)))  ) )  & 
             []( ((!s.bit0 & s.bit1 & s.bit2)) -> ( ((!next(s.bit0) & next(s.bit1) & next(s.bit2)))
                                    | ((!next(s.bit0) & !next(s.bit1) & !next(s.bit2))) 
                                    | ((!next(s.bit0) & !next(s.bit1) & next(s.bit2))) 
                                    | ((next(s.bit0) & !next(s.bit1) & next(s.bit2)))  ) )  & 
             []( ((s.bit0 & !s.bit1 & !s.bit2)) -> ( ((next(s.bit0) & !next(s.bit1) & !next(s.bit2)))
                                    | ((!next(s.bit0) & !next(s.bit1) & !next(s.bit2))) 
                                    | ((!next(s.bit0) & !next(s.bit1) & next(s.bit2))) 
                                    | ((!next(s.bit0) & next(s.bit1) & !next(s.bit2)))  ) )  & 
             []( ((s.bit0 & !s.bit1 & s.bit2)) -> ( ((next(s.bit0) & !next(s.bit1) & next(s.bit2)))
                                    | ((!next(s.bit0) & next(s.bit1) & !next(s.bit2))) 
                                    | ((!next(s.bit0) & next(s.bit1) & next(s.bit2)))  ) ) 
    );
```

**Comment by cfinucane** [comment]
next() is not currently permitted in goals, since they represent states and not transitions. :dog2:

**Comment by ConstantineLignos** [comment]
Underlying issue fixed in PennNLP/SLURP@0dcfc260ceca10f5c044c93f44d8d78b7e337d2f
